packages/components/src/SubHeaderBar/TitleSubHeader/TitleSubHeader.component.js

```diff
@@ -109,15 +109,22 @@ function SubTitle({ subTitleLoading, subTitle }) {
 	}
 
 	if (subTitle) {
+		let subtitleClassnames;
+		if (asLabel) {
+			subtitleClassnames = classNames('label', 'tc-subheader-details-text-subtitle-label', {
+				'label-info': labelType === 'INFO',
+			});
+		} else {
+			subtitleClassnames = classNames(
+				theme['tc-subheader-details-text-subtitle'],
+				'tc-subheader-details-text-subtitle',
+			);
+		}
```

Comment:
How do this part could render with conditional properties using classnames api instead of having the asLabel if ?

Reply:
I tried, but it was not really sexy to have all classes as conditionnal properties with `!asLabel` or `asLabel`. "label" classes are not an addition to the original style, they replace them.

If you see a sexy way to write with classname, i'll be glad to use it :)

Reply:
This part has been rewritten